core/src/main/java/org/infinispan/container/versioning/SimpleClusteredVersion.java

```diff
@@ -24,14 +25,13 @@
    /**
     * The cache topology id in which it was first created.
     */
-   @ProtoField(number = 1, defaultValue = "0")
-   int topologyId;
+   @ProtoField(number = 1, defaultValue = "-1")
+   final int topologyId;
```

Comment:
topologyId and version fields also have getters. In that case it is better to annotate the getter and make the field private.